tests/test_editor.py

```diff
@@ -291,5 +302,15 @@ class TestSettings:
         QTest.keyClicks(editor, 'World')
         self.assertEqual(editor.document().toPlainText(), '1. Hello\n1. World')
 
+    def test_markdown_list_indentation_normalization(self):
```

Comment:
Thank you very much for the updates!

This test is good, but does it really test your `_normalizeMarkdownListIndentation` function? It looks like no HTML to markdown conversion is happening here, and this test is testing the editor functionality of preserving indentation instead?

If I'm right, you may keep this test and add another one. Or, maybe just add more complex content (that requires normalization) to your `test_pasteHtmlConvertedToMarkdown` function above?

Also, please rename this test to camelCase if you keep it.